docs/reference/errors-and-warnings/NU1605.md

```diff
@@ -62,4 +62,8 @@ To work around this issue, add the following PackageReference:
 The mvc project specified a version constraint on a higher version of a package than restore ultimately resolved. That is because of the [nearest-wins](../../concepts/dependency-resolution.md#nearest-wins) rule - when resolving packages, the version of the nearer package in the graph will override that of the distant package with the same ID.
 
 ### Solution
-This specific error (with Microsoft.NETCore.App package) is improved my moving your .NET Core SDK to 2.2.100 or later. Microsoft.NETCore.App is an auto-referenced package that the .NET Core SDK before version 3.0.100 chooses to bring in automatically. Also see [Self-contained deployment runtime roll forward](/dotnet/core/deploying/runtime-patch-selection).
+This specific error (with Microsoft.NETCore.App package) is improved by moving your .NET Core SDK to 2.2.100 or later. Microsoft.NETCore.App is an auto-referenced package that the .NET Core SDK before version 3.0.100 chooses to bring in automatically. Also see [Self-contained deployment runtime roll forward](/dotnet/core/deploying/runtime-patch-selection).  
+
+You can also try to manually update the packages that are giving you the downgrade error.   
```

Comment:
This shouldn't be true.

Packages marked as "autoreferenced" cannot be updated in VS. 
Auto referenced packages should be only be updated with the SDK as there's no guarantee the SDK you have installed will work with a newer version of the auto refed package.